src/translations/sqlb_de.ts

```diff
@@ -2393,7 +2393,7 @@ x~y	Bereich: Werte zwischen x und y
     <message>
         <location filename="../FindReplaceDialog.ui" line="96"/>
         <source>When set, the search goes backwards from cursor position, otherwise it goes forward</source>
-        <translation>Falls gesetzt, erfolgt die Suche rückwärts von der Cursorposition, andernfalls erfolgt sie vorwärts</translation>
+        <translation>Wenn gesetzt, erfolgt die Suche rückwärts von der Cursorposition, anderenfalls erfolgt sie vorwärts</translation>
```

Comment:
Is the second change really necessary. I personally have never used/seen the new word before, although being a native speaker.